hypothesis-python/tests/cover/test_reflection.py

```diff
@@ -622,3 +623,33 @@ def f(source):
 
     with pytest.raises(ValueError, match="SyntaxError because `from` is a keyword"):
         get_signature(f)
+
+
+def test_param_is_called_within_func():
+    def f(any_name):
+        any_name()
+
+    assert is_func_param_called_within(f, "any_name")
+
+
+def test_param_is_called_within_func_after_assignment():
```

Comment:
It's OK if this is deprecated; I'd rather check that it's called than allow reassignment.